### DISCUSSION TITLE
How I can animate ?

### DISCUSSION THREAD

**Discussion by RetroAndDev** [CosmosOS/Cosmos#1608] [Q&A]
Hello, I would like to animate my OS (with vertical / horizontal transformations or fades) 
How should I go about it?
Thanks for your help

**Comment by quajak** [comment]
Are you working with a text based or graphics based OS?

**Reply by RetroAndDev** [reply]
I use the graphics system

**Reply by quajak** [reply]
There is no built in support for animations, so you will have to implement them yourself.
